### PR TITLE
LPS-172251 Remove Stream and Optional usages in portal-workflow-kaleo-runtime-integration-impl

### PR DESCRIPTION
**Pull request by julschong**

### CODE REVIEW

File: modules/apps/portal-workflow/portal-workflow-kaleo-runtime-integration-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/integration/internal/WorkflowTaskManagerImpl.java [hunk 1]
```diff
@@ -312,15 +310,15 @@ public List<String> getNextTransitionNames(long userId, long workflowTaskId)
 
 			KaleoNode kaleoNode = kaleoTask.getKaleoNode();
 
-			return Stream.of(
-				kaleoNode.getKaleoTransitions()
-			).flatMap(
-				List::parallelStream
```
Comment (julschong) [inline modules/apps/portal-workflow/portal-workflow-kaleo-runtime-integration-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/integration/internal/WorkflowTaskManagerImpl.java:318]: @tinatian Hi Tina,
Do I need to do anything different to convert parallel stream?
Online information seems to suggests that for loop is faster than both serial and parallel stream

Comment (julschong) [reply modules/apps/portal-workflow/portal-workflow-kaleo-runtime-integration-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/integration/internal/WorkflowTaskManagerImpl.java:318]: https://blogs.oracle.com/javamagazine/post/java-parallel-streams-performance-benchmark

**Comment by catarineaguiar** [comment]
no regressions found.

**Comment by tinatian** [comment]
See https://github.com/liferay-core-infra/liferay-portal/pull/2982

**Pull request closed without merging**